docs/lms_config.ini

```diff
@@ -32,10 +31,17 @@ progress = SFD_check_progress.html
 
 learneraccountsettings = sfd_dashboard_profile/SFD_dashboard_settings_profile.html#exploring-the-account-settings-page
 
-learnerdashboard = sfd_dashboard_profile/SFD_dashboard_settings_profile.html#exploring-the-learner-dashboard
+learnerdashboard = sfd_dashboard_profile/SFD_dashboard_settings_profile.html#exploring-the-dashboard
+
+programs = SFD_enrolling.html#programs
```

Comment:
This link results in the maze for the Open edX learner's guide. Shouldn't be a problem if no one in Open edX land can navigate to this page.